### PR TITLE
Improving previous skeleton for README of evolution pallet

### PR DESCRIPTION
**Pull request by tonimateos**

### CODE REVIEW

File: ownership-chain/pallets/laos-evolution/README.md [hunk 1]
```diff
@@ -1 +1,18 @@
...
\ No newline at end of file
+# LAOS Evolution Pallet
+
+## Overview
+The LAOS Evolution Pallet is a core component of LAOS, providing a framework for managing the minting and evolution of digital assets while allowing the management of their ownership to be governed by external consensus systems.
+
+## Installation and Setup
+
+```
+$ git clone https://github.com/freeverseio/laos.git
```
Comment (dastanfv) [inline ownership-chain/pallets/laos-evolution/README.md:9]: this stuff should be in the main README, I think it's ok to remove it or just leave `cargo test`

File: ownership-chain/pallets/laos-evolution/README.md [hunk 2]
```diff
@@ -1 +1,18 @@
-License: MIT-0
\ No newline at end of file
+# LAOS Evolution Pallet
+
+## Overview
+The LAOS Evolution Pallet is a core component of LAOS, providing a framework for managing the minting and evolution of digital assets while allowing the management of their ownership to be governed by external consensus systems.
+
+## Installation and Setup
```
Comment (dastanfv) [inline ownership-chain/pallets/laos-evolution/README.md:6]:
```suggestion
## Tests
```

File: ownership-chain/pallets/laos-evolution/README.md (hunk 1 above)
Comment (tonimateos) [reply ownership-chain/pallets/laos-evolution/README.md:9]: Agreed, just left the cargo test

File: ownership-chain/pallets/laos-evolution/README.md (hunk 2 above)
Comment (tonimateos) [reply ownership-chain/pallets/laos-evolution/README.md:6]: Agreed, changed.

**Review by magecnion — APPROVED** [review]

File: ownership-chain/pallets/laos-evolution/README.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
\ No newline at end of file
+# LAOS Evolution Pallet
+
+## Overview
+The LAOS Evolution Pallet is a core component of LAOS, providing a framework for managing the minting and evolution of digital assets while allowing the management of their ownership to be governed by external consensus systems.
+
+## Tests
+
+```
+$ cargo build --release
```
Comment (dastanfv) [inline ownership-chain/pallets/laos-evolution/README.md:9]:
```suggestion
```

**Review by dastanfv — APPROVED** [review]

**Pull request merged**